integrationTests/basic/vitest.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    // required to make jest-when work with vitest
+    globals: true,
+    exclude:[
+      ...configDefaults.exclude,
+      './src/**',
+      '**/integrationTests/e2e/**',
+      '**/integrationTests/e2e-tls/**',
+      '**/integrationTests/enterprise/**',
```

Comment:
I wonder if this we can replace excluding these with https://vitest.dev/config/#include in `integrationTests/<test-module>/vitest.config.js` files.
```
exclude:[
      ...configDefaults.exclude,
     './src/**',
     '**/integrationTests/**',
],
include: [
'**/integrationTests/basic/**',
]
```
I think it'll require a little more effort to maintain if we have new test modules under `integrationTests`.

Sorry I might be missing the difference between the `exclude` and `include` approaches here as I'm picking up `vitest`. Let me know your thought.